eo-maven-plugin/src/main/java/org/eolang/maven/tojos/ForeignTojo.java

```diff
@@ -404,4 +404,22 @@ public boolean equals(final Object other) {
     public int hashCode() {
         return Objects.hash(this.delegate);
     }
+
+    /**
+     * Return the attribute from the tojo.
+     * @param attribute The attribute from ForeignTojos.Attribute.
+     * @return The attribute.
+     */
+    private String attributeTojo(final ForeignTojos.Attribute attribute) {
```

Comment:
@Yanich96 let's rename it with just `attribute`

Reply:
@maxonfjvipon I fixed